docs/docs/ml.xml

```diff
@@ -817,6 +818,28 @@ Davis E. King. <a href="http://jmlr.csail.mit.edu/papers/volume10/king09a/king09
          </description>
       </component>
 
+
+   <!-- ************************************************************************* -->
+
+      <component>
+         <name>input_rgb_image_pair</name>
+         <file>dlib/dnn.h</file>
+         <spec_file link="true">dlib/dnn/input_abstract.h</spec_file>
+         <description>
+            This is a simple input layer type for use in a deep neural network
+            which takes a pair of RGB images as input and loads it into a network.
+            It is useful when you want to input image pairs that are related to each other,
+            for instance, they are different distorted views of the same original image.
+            This input layer is meant to be used with a loss layer such as the
+            <a href="#loss_barlow_twins_">Barlow Twins loss layer</a>.  You can also
```

Comment:
There would need to be a block on ml.xml for `#loss_barlow_twins_` to link to anything.  So you need to add one or link directly to the abstract file.

Reply:
done!